app/Resources/views/promotional-collection.html.twig

```diff
@@ -8,6 +8,16 @@
 
     {{ render_pattern(contentHeader) }}
 
+    <div class="wrapper">
+
+        {% if contextualData %}
```

Comment:
Don't think it needs this, as contextual data should never appear here. Suggest replacing with a server-side code comment stating why this wrapper hack is needed.